pkg/driver/krun/krun_darwin.go

```diff
@@ -0,0 +1,121 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package krun
```

Comment:
I'm not sure about the name. The program we use here is named krunkit, and the library is called libkrun. In minikube the driver is called krunkit similar to other drivers (vfkit, qemu). In lima we call the qemu driver qemu, so this driver should call krunkit for consistency.

Reply:
The "Kit" suffix is carried over from Docker, where it was HyperKit and LinuxKit and BuildKit and so on